vis-registers.c

```diff
@@ -262,23 +274,24 @@ Array vis_register_get(Vis *vis, enum VisRegister id) {
 }
 
 const RegisterDef vis_registers[] = {
-	[VIS_REG_DEFAULT]    = { '"', VIS_HELP("Unnamed register")                                 },
-	[VIS_REG_ZERO]       = { '0', VIS_HELP("Yank register")                                    },
-	[VIS_REG_1]          = { '1', VIS_HELP("1st sub-expression match")                         },
-	[VIS_REG_2]          = { '2', VIS_HELP("2nd sub-expression match")                         },
-	[VIS_REG_3]          = { '3', VIS_HELP("3rd sub-expression match")                         },
-	[VIS_REG_4]          = { '4', VIS_HELP("4th sub-expression match")                         },
-	[VIS_REG_5]          = { '5', VIS_HELP("5th sub-expression match")                         },
-	[VIS_REG_6]          = { '6', VIS_HELP("6th sub-expression match")                         },
-	[VIS_REG_7]          = { '7', VIS_HELP("7th sub-expression match")                         },
-	[VIS_REG_8]          = { '8', VIS_HELP("8th sub-expression match")                         },
-	[VIS_REG_9]          = { '9', VIS_HELP("9th sub-expression match")                         },
-	[VIS_REG_AMPERSAND]  = { '&', VIS_HELP("Last regex match")                                 },
-	[VIS_REG_BLACKHOLE]  = { '_', VIS_HELP("/dev/null register")                               },
-	[VIS_REG_CLIPBOARD]  = { '*', VIS_HELP("System clipboard register, see vis-clipboard(1)")  },
-	[VIS_REG_DOT]        = { '.', VIS_HELP("Last inserted text")                               },
-	[VIS_REG_SEARCH]     = { '/', VIS_HELP("Last search pattern")                              },
-	[VIS_REG_COMMAND]    = { ':', VIS_HELP("Last :-command")                                   },
-	[VIS_REG_SHELL]      = { '!', VIS_HELP("Last shell command given to either <, >, |, or !") },
-	[VIS_REG_NUMBER]     = { '#', VIS_HELP("Register number")                                  },
```

Comment:
Please avoid changing the code formatting while also introducing logical changes. Either do it in a separate commit, or preferably use a shorter help text.